pipeline/src/transformers/addressables/article.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  asText,
+  asTitle,
+  isFilledLinkToDocumentWithData,
+  isNotUndefined,
+} from '@weco/content-pipeline/src/helpers/type-guards';
+import { ArticlePrismicDocument } from '@weco/content-pipeline/src/types/prismic';
+import { ElasticsearchAddressableArticle } from '@weco/content-pipeline/src/types/transformed';
+
+export const transformAddressableArticle = (
+  document: ArticlePrismicDocument
+): ElasticsearchAddressableArticle => {
+  const { data, id, uid: documentUid } = document;
+
+  const primaryImage = data.promo?.[0]?.primary;
+  const description = primaryImage?.caption && asText(primaryImage.caption);
+  const title = asTitle(data.title);
+  const uid = documentUid || undefined;
+
+  const format = isFilledLinkToDocumentWithData(data.format)
+    ? asText(data.format.data.title)
+    : undefined;
+
+  const contributors = (data.contributors ?? [])
+    .map(c => {
+      return isFilledLinkToDocumentWithData(c.contributor)
+        ? asText(c.contributor?.data.name)
+        : undefined;
+    })
+    .filter(isNotUndefined)
+    .join(', ');
+
+  const queryBody = data.body
+    ?.map(slice => {
+      if (['text', 'quote', 'standfirst'].includes(slice.slice_type)) {
+        return slice.primary.text.map(text => text.text);
+      } else {
+        return [];
+      }
+    })
+    .flat();
```

Comment:
I copied this from what we had in the base articles transformer, because we didn't need the content from embeds, or image galleries, etc. Just these three slices.